client/src/appsupport.js

```diff
@@ -247,6 +247,10 @@ window.Dark = {
     busy: false,
     /* Records the last time a result returned. So Integration tests will know has analysis finished running since a given timestamp */
     utils: require("../../lib/js/client/workers/FSharpAnalysisWrapper.bs.js"),
+    debug: (function () {
```

Comment:
note: this is within the fsharpAnalysis. Not bothering to make this work with the ocaml stuff since we'll be deleting that soon anyway.